cmd/notary/main.go

```diff
@@ -37,14 +37,11 @@ func parseConfig() {
 	}
 
 	if trustDir == "" {
-		// Retrieve current user to get home directory
-		usr, err := user.Current()
+		// Get home directory for current user
+		homeDir, err := homedir.Dir()
 		if err != nil {
-			fatalf("cannot get current user: %v", err)
+			fatalf("cannot get current user home directory: %v", err)
 		}
-
-		// Get home directory for current user
-		homeDir := usr.HomeDir
 		if homeDir == "" {
```

Comment:
`go-homedir` returns an `err` if it can't find a homedir. This check can safely be removed.